coralproject/talk
# [CORL-1241] Analytics Bugs

## What does this PR do?

When a `userId` is not available during an analytics event (such as when a story is created via lazy story creation), it should use the request trace id as the `anonymousId` on the event.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?

On a clean install, with the analytics environment variables set, visit the development story page and see the analytics event emitted successfully. You might need to set `LOGGING_LEVEL=trace` to see those log messages.
